### PR TITLE
Add readme update for running in development

### PR DESCRIPTION
**Pull request by linqiu**
I was following the readme to start running the app. This part was not apparent to me, nor was it easy to find via reddit and searching. So I figured my first PR would be to update the readme with this info :)

### CODE REVIEW

**Comment by linqiu** [comment]
@Hucxley I was thinking about this, it seems a bit high risk for beginners to know that when the run the app, it would actually run against comments on reddit. 

I will certainly in the short term add the flags notes in readme. But do we have have an issue to possible set those two things to always be true when we're running in dev/test? I think in the medium term that would actually alleviate and lower the activation energy required to run this app?

**Comment by joegoldbeck** [comment]
@linqiu actually, it _is_ safe to run by default for exactly this reason :D 

No replies will be sent unless you explicitly set `--send-replies`. (Please do not set this unless you are 1000% sure you're in a sandbox subreddit)

In fact, you can safely leave out `--simulate-replies`. When combined with `--skip-tracking`, it doesn't actually _do_ anything except to change the print statements slightly. The primary use of `--simulate-replies` would be to update your local database as if you _had_ replied, but with `--skip-tracking` you won't interface with a local db anyway!

File: README.md [hunk 1]
```diff
@@ -125,7 +129,7 @@ Strategies are defined as static methods of the Strategy class in `matching.py`
 - Without making actual replies
 - Without checking a posts_replied_to list
 
-`python src/main.py --skip-tracking`
+`python src/main.py --skip-tracking --simulate-replies`
```
Comment (joegoldbeck) [inline README.md]: ` --simulate-replies` is unneeded :D 

As noted above, you won't make replies with `python src/main.py --skip-tracking`

File: README.md [hunk 2]
```diff
@@ -16,6 +16,10 @@ The bot can be summoned by making a comment of the form `!WarrenPlanBot plan_top
 
 You'll need a `praw.ini` file in the '/src' folder of this repo. Copy over the `praw.ini.example` file and fill in the details. You'll need to get the client_id and client_secret from the owners of the project.
 
+In order to test in development, you will need to [create your own reddit dev app](https://github.com/reddit-archive/reddit/wiki/OAuth2-Quick-Start-Example#first-steps). Then use the credentials to populate your `praw.ini` file for dev.
+
+Also please see how to [run the app in dev safely](#run-the-bot)
```
Comment (joegoldbeck) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -16,6 +16,10 @@ The bot can be summoned by making a comment of the form `!WarrenPlanBot plan_top
 
 You'll need a `praw.ini` file in the '/src' folder of this repo. Copy over the `praw.ini.example` file and fill in the details. You'll need to get the client_id and client_secret from the owners of the project.
 
+In order to test in development, you will need to [create your own reddit dev app](https://github.com/reddit-archive/reddit/wiki/OAuth2-Quick-Start-Example#first-steps). Then use the credentials to populate your `praw.ini` file for dev.
```
Comment (joegoldbeck) [inline README.md]: 👍

**Pull request merged**